rmgpy/kinetics/arrhenius.pyx

```diff
@@ -152,6 +152,13 @@ cdef class Arrhenius(KineticsModel):
         data.
         """
         import scipy.stats
+        assert all(np.isfinite(klist)), "Rates must all be finite, not inf or NaN"
+        if any(klist<0):
+            assert all(klist<0), "Rates must all be positive or all be negative."
```

Comment:
Could you convert these assertions to `raise ValueError` instead?

Reply:
Done. as a fixup, when rebasing to latest master.